eventmesh-dashboard-common/pom.xml

```diff
@@ -40,6 +40,17 @@
             <artifactId>fastjson2</artifactId>
             <version>2.0.40</version>
         </dependency>
+        <!-- Event Store -->
+        <dependency>
+            <groupId>org.apache.rocketmq</groupId>
+            <artifactId>rocketmq-client</artifactId>
+            <version>4.9.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.rocketmq</groupId>
+            <artifactId>rocketmq-tools</artifactId>
+            <version>4.9.5</version>
+        </dependency>
```

Comment:
Better use a unified variable to store RocketMQ's version. By the way, the latest version is 4.9.8.